gradle-module/src/main/java/org/example/Main.java

```diff
@@ -22,4 +23,11 @@ public static void main(String[] args) {
     private static void test() {
         System.out.println("test");
     }
-}
\ No newline at end of file
+
+   /* asd
+    we
+    asd
+
+    */
+    
```

Comment:
The comment block at the end of the file appears to be a placeholder or incomplete. It would be better to either remove it or replace it with meaningful documentation that explains the purpose of the class or methods. This will improve maintainability and clarity.